packages/url_launcher/url_launcher_linux/example/pubspec.yaml

```diff
@@ -26,3 +26,8 @@ dev_dependencies:
 
 flutter:
   uses-material-design: true
+
+# FOR TESTING AND INITIAL REVIEW ONLY. DO NOT MERGE.
+# See https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changing-federated-plugins
+dependency_overrides:
+   {url_launcher_platform_interface: {path: ../../../url_launcher/url_launcher_platform_interface}}
```

Comment:
Here and elsewhere, I assume this will be backed out?

Reply:
Yes, this is the combo PR. Once it's approved I'll break it into three and land then in order, and these become constraint updates on the new version.